README.md

```diff
@@ -53,31 +57,32 @@ and [read our docs for more information about indexing your data](https://www.fi
 
 ## Including GeoFire in your project Android/Java
 
-In order to use GeoFire in your project, you need to [add the Firebase Java
-SDK](https://www.firebase.com/docs/java-quickstart.html?utm_source=geofire-java).
-There are then multiple possibilities to use GeoFire in your project.
+In order to use GeoFire in your project, you need to [add the Firebase Android
+SDK](https://firebase.google.com/docs/android/setup). After that you can include
+GeoFire with one of the choices below.
+
+### Gradle
+
+Add a dependency for GeoFire to your `gradle.build` file:
+
+```groovy
+dependencies {
+    compile 'com.firebase:geofire:2.0.0+'
```

Comment:
nit: can we discourage the '+' here and elsewhere?  It slows down gradle builds and makes them non-deterministic.